examples/typescript/src/utils/election-process.ts

```diff
@@ -0,0 +1,63 @@
+import chalk from 'chalk';
+import { getDefaultClient, submitVote, waitForElectionReady } from './utils';
+import { Wallet } from '@ethersproject/wallet';
+import { Election, IVoteType, OffchainCensus, UnpublishedElection } from '@vocdoni/sdk';
+
+export const createElection = (census: OffchainCensus, electionOpts: IVoteType, title: string, description: string) => {
+  const endDate = new Date();
+  endDate.setHours(endDate.getHours() + 10);
+
+  const election: UnpublishedElection = Election.from({
+    title: title + ' ' + endDate.toISOString(),
+    description: description,
+    header: 'https://source.unsplash.com/random',
+    streamUri: 'https://source.unsplash.com/random',
+    endDate: endDate.getTime(),
+    census,
+    voteType: electionOpts,
+  });
+
+  return election;
+};
+
+export const executeElection = async (election: UnpublishedElection, participants: Wallet[], voteArray: number[]) => {
+  console.log(chalk.yellow('Creating a new quadratic voting process!'));
```

Comment:
This is used several times, therefore the message is confusing because we are not creating a quadratic voting everytime

Reply:
Fixed afb06f1fc3ef96e050c77a8a10e43199b6e7cd5b